account_statement_import_online_plaid/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Binhex - Adasat Torres de León.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Online Bank Statements: plaid.com",
+    "version": "16.0.1.0.0",
+    "category": "Account",
+    "website": "https://github.com/OCA/bank-statement-import",
+    "author": "Binhex, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": ["account_statement_import_online"],
+    "data": [
+        "views/online_bank_statement_provider.xml",
+    ],
+    "assets": {
+        "web.assets_backend": [
+            "/account_statement_import_online_plaid/static/src/**/*.js",
+            "https://cdn.plaid.com/link/v2/stable/link-initialize.js",
```

Comment:
It is prefered to download the library locally in order to avoid external connections

Reply:
Yes, please put a copy. Systems may be isolated and don't access any external resource. Also for security/privacy policies.

Reply:
Hi, I understand, but on Plaid documentation indicate to use the CDN link because they haven't any local library. https://plaid.com/docs/link/web/.
I download the file with wget but when add the file route on backend assets odoo stop working.
![image](https://github.com/OCA/bank-statement-import/assets/128601880/b101050c-3585-4714-a44b-c99a0a48839a)

Reply:
Maybe there are extra files that are imported in that main file? @etobella can you give here a hand?

Reply:
They ask for cdn because it is easier, but we can download the code from https://cdn.plaid.com/link/v2/stable/link-initialize.js and copy it directly to a JS, I don't see the issue there.

where you using debug=assets? Sometimes it gives better errors on JS